### PR TITLE
Home Assistant | ObscuraCam | Audio Spectrum Analyzer | Liedgutverzeichnis | Tux Paint | OpenKeychain | Molly

### PR DESCRIPTION
**Pull request by shuvashish76**
Neo-Store - Droid-ify renamed to Neo-Store (repo link changed) F-Droid will update the name from upcoming releases
K-9 Mail - repo link changed (https://k9mail.app/2022/06/13/K-9-Mail-and-Thunderbird.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,6 +347,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
     * [**Forkgram**](https://github.com/Forkgram/TelegramAndroid) <sup>**[[F-Droid](https://f-droid.org/app/org.forkgram.messenger)]**</sup>
 * [**Element**](https://github.com/vector-im/element-android) <sup>**[[F-Droid](https://f-droid.org/app/im.vector.app)]**</sup>
 * [**Signal**](https://github.com/signalapp/Signal-Android)
+    * [**Molly**](https://github.com/mollyim/mollyim-android)
```
Comment (offa) [inline README.md:350]: It seems to provide a Free build flavours, which exclude GMS, Play, Firebase etc. :+1:

File: README.md [hunk 2]
```diff
@@ -433,15 +438,17 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
...
 ### • Password & Authentication
 
 * [**FreeOTP+**](https://github.com/helloworld1/FreeOTPPlus) <sup>**[[F-Droid](https://f-droid.org/app/org.liberty.android.freeotpplus)]**</sup>
 * [**andOTP**](https://github.com/andOTP/andOTP) <sup>**[[F-Droid](https://f-droid.org/app/org.shadowice.flocke.andotp)]**</sup>
 * [**KeePassDX**](https://github.com/Kunzisoft/KeePassDX) <sup>**[[F-Droid](https://f-droid.org/app/com.kunzisoft.keepass.libre)]**</sup>
 * [**Aegis**](https://getaegis.app/) <sup>**[[F-Droid](https://f-droid.org/app/com.beemdevelopment.aegis)]**</sup>
-* [**Bitwarden Mobile**](https://github.com/bitwarden/mobile)
+* [**Bitwarden**](https://github.com/bitwarden/mobile)
 * [**AuthPass**](https://github.com/authpass/authpass) <sup>**[[F-Droid](https://f-droid.org/app/design.codeux.authpass.fdroid)]**</sup>
+* [**OpenKeychain**](https://github.com/open-keychain/open-keychain) <sup>**[[F-Droid](https://www.f-droid.org/app/org.sufficientlysecure.keychain)]**</sup>
```
Comment (offa) [inline README.md:451]: It's no longer actively developed according to it's readme, but they still to basic maintenance, merge PRs etc.

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Thanks! :-)

File: README.md (hunk 2 above)
Comment (shuvashish76) [reply README.md:451]: 👍🏿 v5.8.0 added to F-Droid on 05-01-2023

**Pull request merged**